### DISCUSSION TITLE
[ Laravel AI ] Ideas: AI LLM-based RAG package supported by Laravel framework.

### DISCUSSION THREAD

**Discussion by qwerty199369** [laravel/framework#53458] [Ideas]
[ Laravel AI ] Given AI LLM-based development has become a new trend, does the Laravel community have plans to launch a LLM RAG package supported by Laravel framework, just like LLPhant?

https://github.com/theodo-group/LLPhant

**Comment by charescape** [comment]
Yeah, our new project is indeed ai-first and based on LLPhant.

**Reply by charescape** [reply]
Looking forward to laravel also launching ai development related packages.

**Comment by shaedrich** [comment]
Just because something is a trend, not everyone has to jump the bandwagon. While there are _some_ use cases for AI, not every software has to have _their_ very own AI feature just because it looks shiny, let alone, when it's not the use case for AI, AI can't improve upon a non-AI implementation. Not to mention that most AIs depend on theft, so there's the moral problem with it. Also, even though, Laravel has a wide variety of features, I don't think, AI is something that is _needed_ in its core. This would work better as a package.

**Comment by charescape** [comment]
A unified interface for working with LLMs in Laravel: https://github.com/echolabsdev/prism
